Sources/TuistDependencies/Carthage/Utils/CarthageGraphGenerator.swift

```diff
@@ -28,7 +28,10 @@ public final class CarthageGraphGenerator: CarthageGraphGenerating {
 
         let thirdPartyDependencies: [String: ThirdPartyDependency] = Dictionary(grouping: products, by: \.name)
             .compactMapValues { product in
-                guard let frameworkName = product.first?.container else { return nil }
+                // All the products grouped by name have the same name and framework
+                guard let firstProduct = product.first else { return nil }
```

Comment:
nit: I think this can be called `product` and the name for the closure parameter should rather be `products` (`product` implies it should be just one but there can be multiple of them)

Reply:
Changed `product` to `products`, but I would keep this as `firstProduct` to make clear that there can be more than one